packages/apps/graph-client/src/components/chainweb/chain-block/chain-block.tsx

```diff
@@ -14,7 +14,7 @@ interface IChainBlockProps {
   textColor: string;
 }
 
-export function ChainBlock(props: IChainBlockProps): JSX.Element {
+export const ChainBlock = (props: IChainBlockProps): JSX.Element => {
```

Comment:
Do we maybe prefer this `const Component: React.FC<IChainBlockProps> = (props) => {}` over components having types separately for props and return value?

If so, it'd be nice if there's a way to auto-fix this construct.

Reply:
Nothing in this PR is necessarily a preference, things like this are all auto-fixed based on existing configuration.